services/QuillLMS/client/app/bundles/PremiumHub/components/subnav_tabs.tsx

```diff
@@ -92,8 +111,31 @@ export default class AdminSubnav extends React.Component<any, any> {
   }
 
   render() {
-    const { overview, schoolSubscriptions, activityScores, conceptReports, standardsReports, dropdownOpen, usageSnapshotReport, dataExport, activeTab } = this.state
-    const activeStates = [overview, schoolSubscriptions, activityScores, conceptReports, standardsReports, dataExport, usageSnapshotReport]
+    const {
+      activeTab,
+      activityScores,
+      conceptReports,
+      dataExport,
+      dropdownOpen,
+      integrations,
+      overview,
+      schoolSubscriptions,
+      standardsReports,
+      usageSnapshotReport,
+    } = this.state
+
+    // Order here is coupled to subnav_tab layout
```

Comment:
Thanks for that comment.

Reply:
This threw me off for an hour when I added integrations and then alphabetized the array.